pkg/systemd/host.css

```diff
@@ -5,6 +5,10 @@
 @import "../lib/plot.css";
 @import "../lib/table.css";
 @import "../lib/tooltip.css";
+@import "../lib/cockpit-components-empty-state.css";
+@import "../../node_modules/@patternfly/patternfly/components/Button/button.css";
```

Comment:
OK I just tried it without these css imports and indeed it looks it's needed.
But still I am really concerned why - each component should be able to handle it's own css imports.

Reply:
In any case if we really need to import these I think we should do that in the pkg/lib/cockpit-components-empty-state.css instead.

Reply:
yeah, this is a bit stupid, but the problem is that this creates `dist/systemd/logs.css` but `logs.css` is not loaded. I could add it into `logs.html` and then figure out how to package it as well (I guess somewhere in webpack config). But since logs were using `host.css` I just added it there.

Reply:
``` diff
$ git diff
diff --git a/pkg/systemd/host.css b/pkg/systemd/host.css
index f0700f0fe..ab3af68a5 100644
--- a/pkg/systemd/host.css
+++ b/pkg/systemd/host.css
@@ -6,9 +6,6 @@
 @import "../lib/table.css";
 @import "../lib/tooltip.css";
 @import "../lib/cockpit-components-empty-state.css";
-@import "../../node_modules/@patternfly/patternfly/components/Button/button.css";
-@import "../../node_modules/@patternfly/patternfly/components/Spinner/spinner.css";
-@import "../../node_modules/@patternfly/patternfly/components/EmptyState/empty-state.css";
 
 @import "./timer.css";
 
diff --git a/pkg/systemd/logs.html b/pkg/systemd/logs.html
index 64fd7d39d..f60d60660 100644
--- a/pkg/systemd/logs.html
+++ b/pkg/systemd/logs.html
@@ -24,6 +24,7 @@ along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
   <meta charset="utf-8">
   <link href="../base1/patternfly.css" rel="stylesheet">
   <link href="system.css" rel="stylesheet">
+  <link href="logs.css" rel="stylesheet">
   <script type="text/javascript" src="../base1/jquery.js"></script>
   <script type="text/javascript" src="../base1/cockpit.js"></script>
   <script src="../*/po.js"></script>
```

This works and I think is cleaner than what you have now.